### PR TITLE
Ac 573

### PR DESCRIPTION
**Pull request by crimsonsunset**
### Checklist for Pull Requests

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [ ] Make sure your branch matches the naming conventions. (Something like: `AC-152`). Don't request your master!
- [ ] Your branch should only contain changes relevant to the ticket it is named after.
- [ ] Make sure you are making a pull request against the **dev branch** (left side). Also you should start _your branch_ off _our dev_.
- [ ] Check your code additions will fail neither code linting checks nor e2e test (this will be tested automatically when you open your PR).

### Description

Please describe anything in your pull request that isn't immediately understandable from the ticket description.

Thank you! 😎

### CODE REVIEW

File: src/components/map-components/google-map.js [hunk 1]
```diff
@@ -73,6 +76,24 @@ class GoogleMap extends Component {
       latitude = get(resp, 'data.latitude');
       longitude = get(resp, 'data.longitude');
       selection = resp.data;
+    } else if (state) {
+      // state in url
+
+      // check if its actually a state, it will change the necessary search term
+      const searchTerm = (NON_STATES.includes(state.toLowerCase())) ? state : `state of ${state}`;
+      const results = await geocodeByAddress(searchTerm)
```
Comment (bparrish17) [inline src/components/map-components/google-map.js:84]: worried about empty results error here... can we catch and return an empty array?

Comment (crimsonsunset) [reply src/components/map-components/google-map.js:84]: the isError will catch it actually, you can try it out by doing /map/asdasdasasdasd

Comment (crimsonsunset) [reply src/components/map-components/google-map.js:84]: image

Comment (crimsonsunset) [reply src/components/map-components/google-map.js:84]: image

**Comment by crimsonsunset** [comment]
merging per comments

**Pull request merged**